cmd/geranos/main.go

```diff
@@ -43,21 +43,17 @@ func Run(_ []string) error {
 	const sourceRegistry = "us-central1-docker.pkg.dev/k8s-artifacts-prod/images"
 
 	// TODO: make configurable later
-	const s3Bucket = "s3:prod-registry-k8s-io-us-east-2"
+	const s3Bucket = "prod-registry-k8s-io-us-east-2"
 
 	repo, err := name.NewRepository(sourceRegistry)
 	if err != nil {
 		return err
 	}
-	s3Uploader, err := newS3Uploader()
+	s3Uploader, err := newS3Uploader(os.Getenv("REALLY_UPLOAD") == "")
```

Comment:
```suggestion
	s3Uploader, err := newS3Uploader(os.Getenv("DRY_RUN") != "")
```
I'm new here, so maybe this makes more sense to everyone else.

Reply:
The default for now is to do a dry run. This would invert that default.